test/extended/util/framework.go

```diff
@@ -247,60 +247,68 @@ func GetMasterThreadDump(oc *CLI) {
 // ExamineDiskUsage will dump df output on the testing system; leveraging this as part of diagnosing
 // the registry's disk filling up during external tests on jenkins
 func ExamineDiskUsage() {
-	out, err := exec.Command("/bin/df", "-m").Output()
-	if err == nil {
-		e2e.Logf("\n\n df -m output: %s\n\n", string(out))
-	} else {
-		e2e.Logf("\n\n got error on df %v\n\n", err)
-	}
-	out, err = exec.Command("/bin/docker", "info").Output()
-	if err == nil {
-		e2e.Logf("\n\n docker info output: \n%s\n\n", string(out))
-	} else {
-		e2e.Logf("\n\n got error on docker inspect %v\n\n", err)
-	}
+	// disabling this for now, easier to do it here than everywhere that's calling it.
+	return
+	/*
+		out, err := exec.Command("/bin/df", "-m").Output()
+		if err == nil {
+			e2e.Logf("\n\n df -m output: %s\n\n", string(out))
+		} else {
+			e2e.Logf("\n\n got error on df %v\n\n", err)
+		}
+		out, err = exec.Command("/bin/docker", "info").Output()
+		if err == nil {
+			e2e.Logf("\n\n docker info output: \n%s\n\n", string(out))
+		} else {
+			e2e.Logf("\n\n got error on docker inspect %v\n\n", err)
+		}
+	*/
 }
 
 // ExaminePodDiskUsage will dump df/du output on registry pod; leveraging this as part of diagnosing
 // the registry's disk filling up during external tests on jenkins
 func ExaminePodDiskUsage(oc *CLI) {
-	out, err := oc.Run("get").Args("pods", "-o", "json", "-n", "default", "--config", KubeConfigPath()).Output()
-	var podName string
-	if err == nil {
-		b := []byte(out)
-		var list kapiv1.PodList
-		err = json.Unmarshal(b, &list)
+	// disabling this for now, easier to do it here than everywhere that's calling it.
+	return
```

Comment:
@gabemontero fyi, i ended up doing this here for now.  I didn't really want to strip the calls out of all our test cases in case we decide we need it again in the future.

(same for ExamineDiskUsage above)

Reply:
10-4 / sounds good